init.ps1

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env pwsh
+
+Write-Host "`nStarting project setup..." -ForegroundColor Cyan
+
+function Run-Command($cmd, $errorMsg) {
+    try {
+        Write-Host "Running: $cmd" -ForegroundColor Yellow
+        Invoke-Expression $cmd
```

Comment:
`Invoke-Expression` will only throw an exception if the expression throws an exception. If the command fails this will just return a value that is ignored. If you want helpers that throw an exception when a command fails suggest looking at this

https://github.com/dotnet/roslyn/blob/main/eng/build-utils.ps1#L70-L153